plugins/communications-app/TeamEmails/ListTeams.test.jsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import { IntlProvider } from 'react-intl';
+
+import ListTeams from './ListTeams';
+
+describe('ListTeams component', () => {
+  // eslint-disable-next-line no-unused-vars, react/prop-types
```

Comment:
Is this still necessary?

Reply:
'no-unused-vars' isn't necessary, I will remove it. Thanks!